install_config/cluster_metrics.adoc

```diff
@@ -600,25 +600,38 @@ To avoid this issue, use certificates which are configured to be acceptable by
 your browser.
 ====
 
-ifdef::openshift-origin[]
```

Comment:
We need to add a unique anchor here, above the heading, so:

[[cluster-metrics-accessing-hawkular-metrics-directly]]
== Accessing Hawkular Metrics Directly